src/VAC/Scene.cpp

```diff
@@ -810,6 +810,17 @@ SurfaceType Scene::getLoadedSurfaceType()
     return surfaceType;
 }
 
+bool Scene::hasSceneContent() const
+{
+    for (auto layer: layers_)
+    {
```

Comment:
run clang-format, or install in qt creator

Reply:
It's VPaint and it doesn't have clang format, I have installed clang plugin....

Reply:
Or we need to apply clang format to the VPaint too?

Reply:
If I will run clang check on vpaint code - will be a lot of changes...

Reply:
No, I always forget that we don't do that in VPAint.. 

Nevermind... I need more coffee! 😅

Reply:
done